cfgov/paying_for_college/management/commands/update_pfc_national_stats.py

```diff
@@ -0,0 +1,17 @@
+from django.core.management.base import BaseCommand
+
+from paying_for_college.disclosures.scripts import nat_stats
+
+
+COMMAND_HELP = """update_pfc_national_stats gets the latest national statistics
+ yaml file from collegescorecard, parses it and updates our local json file at
+ college/fixtures/national_stats.json.
```

Comment:
Is this another missed paying_for_college -> college -> paying_for_college un-renaming?

Reply:
Indeed it is.